datagram_queue.go

```diff
@@ -4,14 +4,19 @@ import (
 	"context"
 	"sync"
 
-	"github.com/quic-go/quic-go/internal/protocol"
 	"github.com/quic-go/quic-go/internal/utils"
 	"github.com/quic-go/quic-go/internal/wire"
 )
 
+const (
+	maxDatagramSendQueueLen = 32
```

Comment:
Can we make this configurable to let the users experiment with different queue size?

Reply:
I don't really want to add additional API surface here, but if your experimentation finds that this queue is too small, we can increase the queue size.

Reply:
magic environment variables would be useful here. Can label it alpha or beta to let people know it's going away